livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/tts.py

```diff
@@ -43,6 +86,8 @@ def __init__(
         speech_region: str | None = None,
         voice: str | None = None,
         endpoint_id: str | None = None,
+        language: str | None = None,
+        prosody: ProsodyConfig | None = None,
```

Comment:
should we rename this field to "default_prosody"?
I'm just considering the case where users add their own SSML inside the text.

Reply:
Maybe it isn't a big deal since it already defaults to None. So no users will be impacted if they're already using their own SSML

Reply:
@theomonnom also right now it's not possible to use the SSML at all

Reply:
no way to provide SSML using the text, because the Azure TTS calls `synthesizer.speak_text_async(self._text).get()`

Reply:
Ah I see!